config/label-management.md

```diff
@@ -0,0 +1,126 @@
+# Managing cross-organization labels in existing repositories #
+
+This is a step-by-step guide/script for managing organization-wide github
+issue labels across all repositories.
+
+## Prerequisites ##
+
+- User has access to the github organization's repositories
+- User can generate a GitHub [Personal access token](https://github.com/settings/tokens)
+
+## Initial Setup ##
+
+1. Clone the repository locally
+
+```console
+git clone https://github.com/alta3/github-label-management.git
+cd github-label-management
+```
+
+1. Activate the virtual environment and install the requirements
+
+```console
+python3.8 -m venv venv
+source venv/bin/activate
+python -m pip install --upgrade pip wheel
+python -m pip install -r requirements.txt
+```
+
+1. Create a local configuration file by running `touch config`
+1. Open `config` in your favorite editor for editing
+1. Create a GitHub [Personal access token](https://github.com/settings/tokens)
+
+- Give it a descriptive name like "label management"
+- **Only** select the `repo` scope
+- Copy the generated token into `GITHUB_USER_TOKEN`
+
+```sh
+export GITHUB_USER_TOKEN="<YOUR_TOKEN_HERE>"
+export LABELS_TOKEN=${GITHUB_USER_TOKEN}
+export LABELS_USERNAME="<YOUR_USERNAME>"
+export LABELS_ORGNAME="cisagov"
+```
+
+1. Save your changes and run `source config`
+
+## Label management operations ##
+
+There are several operations you'll probably want to do, now that you've
+set up the environment and have the script ready.
+
+### Re-initializing the environment ###
+
+When you return to the directory in a new shell, remember to:
+
+```console
+source venv/bin/activate
+source config
+```
+
+### List all organization repositories ###
+
+```console
+python list-all-repos.py
+```
+
+This will yield a list of all repositories you have access to in the specified
+GitHub organization. If you have not specified a token, this will yield a list
+of all public repositories.
+
+### Back up all current repository labels ###
+
+This will generate a `.toml` file for each repository in a directory of your
+choosing. Each `.toml` file will contain the details of the repository's
+labels, including names, colors, and descriptions.
+
+Please note these output files are the same format as the `default-labels.toml`
+file, so you can use one as a template if you have a repository set up with
+the labels you want organization-wide.
+
+```console
```

Comment:
```suggestion
```sh
```

Reply:
This is command-line usage so I believe `console` is more accurate.

Reply:
After rereading the notes in my forthcoming `style-guide.md`, since this is not showing output and is instead listing commands, I agree that `sh` is more useful for this and following entries.